src/main/protobuf/command_server.proto

```diff
@@ -58,6 +58,8 @@ message RunRequest {
   // came from. These options have already been parsed and already have had
   // their effect. This information should only be used for logging.
   repeated StartupOption startup_options = 6;
+
+  string parameter_file = 7;
```

Comment:
I think this should be a `oneof` with `repeated bytes arg = 2;`, that way it is clear for the java code which field to parse. Right now the java code attempts to parse field 2, if its empty it attempts to parse this.

I'm not sure where this proto file is used, and if this would be backward compatible change. Any tips on how to do this best welcome.